private/data/i18n/en.json

```diff
@@ -642,6 +641,13 @@
           "orderTotal": "Total"
         }
       },
+      "marketplace": {
```

Comment:
Marketplace specific translations could be moved to `marketplace/server/i18n`.   See the payment or taxes plugins for examples of loading translations from there.  Just adding the `en.json` will take care of it, the LingoHub service will translate it into individual language files, but we'll need to manually update the imports once that is done.